Library/ViewModels/PaymentMethodsViewModel.swift

```diff
@@ -155,7 +155,8 @@ public final class PaymentMethodsViewModel: PaymentMethodsViewModelType,
             configuration.allowsDelayedPaymentMethods = true
             let data = PaymentSheetSetupData(
               clientSecret: envelope.clientSecret,
-              configuration: configuration
+              configuration: configuration,
+              paymentSheetType: .setupIntent
```

Comment:
`PaymentMethodsViewModel` is used by `PaymentMethodSettingsViewControllerDelegate`, which is in the settings. That page only ever needs to use a `SetupIntent`.